src/vs/vscode.d.ts

```diff
@@ -1617,7 +1617,7 @@ declare module 'vscode' {
 		/**
 		 * The command this code lens represents.
 		 */
-		command?: Command;
+		command?: Command | string;
```

Comment:
@jrieken I'm not a huge fan of this API design. Let me know if you have any suggestions on a better way to express this idea